ethernet_init/ethernet_init.c

```diff
@@ -277,18 +279,21 @@ static esp_eth_handle_t eth_init_spi(spi_eth_module_config_t *spi_eth_module_con
 #if CONFIG_ETHERNET_USE_KSZ8851SNL
     eth_ksz8851snl_config_t ksz8851snl_config = ETH_KSZ8851SNL_DEFAULT_CONFIG(CONFIG_ETHERNET_SPI_HOST, &spi_devcfg);
     ksz8851snl_config.int_gpio_num = spi_eth_module_config->int_gpio;
-    dev->mac = esp_eth_mac_new_ksz8851snl(&ksz8851snl_config, &mac_config);
-    dev->phy = esp_eth_phy_new_ksz8851snl(&phy_config);
-    sprintf(dev->dev_info.name, "KSZ8851SNL");
+    ksz8851snl_config.poll_period_ms = spi_eth_module_config->poll_period_ms;
+    dev_out->mac = esp_eth_mac_new_ksz8851snl(&ksz8851snl_config, &mac_config);
+    dev_out->phy = esp_eth_phy_new_ksz8851snl(&phy_config);
+    sprintf(dev_out->dev_info.name, "KSZ8851SNL");
 #elif CONFIG_ETHERNET_USE_DM9051
     eth_dm9051_config_t dm9051_config = ETH_DM9051_DEFAULT_CONFIG(CONFIG_ETHERNET_SPI_HOST, &spi_devcfg);
     dm9051_config.int_gpio_num = spi_eth_module_config->int_gpio;
-    dev->mac = esp_eth_mac_new_dm9051(&dm9051_config, &mac_config);
-    dev->phy = esp_eth_phy_new_dm9051(&phy_config);
-    sprintf(dev->dev_info.name, "DM9051");
+    dm9051_config.poll_period_ms = spi_eth_module_config->poll_period_ms;
+    dev_out->mac = esp_eth_mac_new_dm9051(&dm9051_config, &mac_config);
```

Comment:
@bogdankolendovskyy Thanks for the `dev_out` name fix.

@kostaond It would be better if we add build CI with different configurations.

Reply:
@leeebo good idea. @bogdankolendovskyy could you please take care of it?

Reply:
Added.